reference/tools.rst

```diff
@@ -1366,3 +1366,16 @@ Properties:
     - **ar**: Path to archiver (AR).
     - **ranlib**: Path to archive indexer (RANLIB).
     - **strip**: Path to symbol removal utility (STRIP).
+
+tools.latest_visual_studio_version_installed()
+----------------------------------------------
+
+.. code-block:: python
+
+    compiler, version = tools.latest_visual_studio_version_installed(self.output)
```

Comment:
Not sure about returning the compiler here. I don't see the point of calling a function named "latest_**visual_studio**_version_installed" returning a "Visual Studio" compiler

Reply:
I will check the code PR

Reply:
changed in code and doc